compiler/src/dmd/main.d

```diff
@@ -397,7 +398,30 @@ private int tryMain(size_t argc, const(char)** argv, ref Param params)
 
     // Build import search path
 
-    static void buildPath(ref Strings imppath, ref Strings result)
+    static void buildImportPath(ref Array!ImportPathInfo imppath, ref Array!ImportPathInfo result, ref Strings pathsOnlyResult)
```

Comment:
can any of these be `out` instead of `ref`?

Reply:
Possibly, but I don't like the idea of making that change.

Who knows what relies on it.